frontend/src/container/NotificationWall.jsx

```diff
@@ -86,65 +89,167 @@ export class NotificationWall extends React.Component {
-        break
-      case buildTracimLiveMessageEventType(TLM_ET.CONTENT, TLM_CET.MODIFIED, TLM_ST.HTML_DOCUMENT):
-      case buildTracimLiveMessageEventType(TLM_ET.CONTENT, TLM_CET.MODIFIED, TLM_ST.FILE):
-      case buildTracimLiveMessageEventType(TLM_ET.CONTENT, TLM_CET.MODIFIED, TLM_ST.THREAD):
-      case buildTracimLiveMessageEventType(TLM_ET.CONTENT, TLM_CET.MODIFIED, TLM_ST.FOLDER):
-        if (notification.content.currentRevisionType === 'status-update') {
-          icon = 'fa-random'
-          text = props.t('{{author}} updated the status of {{content}} in {{workspace}}', i18nOpts)
-        } else {
-          icon = 'fa-history'
-          text = props.t('{{author}} updated a new version of {{content}} in {{workspace}}', i18nOpts)
-        }
-        url = `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.type}/${notification.content.id}`
-        break
-      case buildTracimLiveMessageEventType(TLM_ET.SHAREDSPACE_MEMBER, TLM_CET.CREATED):
-        icon = 'fa-user-o'
-        text = props.user.userId === notification.user.userId
-          ? props.t('{{author}} added you to {{workspace}}', i18nOpts)
-          : props.t('{{author}} added {{user}} to {{workspace}}', i18nOpts)
-        url = `/ui/workspaces/${notification.workspace.id}/dashboard`
-        break
-      case buildTracimLiveMessageEventType(TLM_ET.MENTION, TLM_CET.CREATED):
-        if (notification.content.type === CONTENT_TYPE.COMMENT) {
-          icon = 'fa-comment-o'
-          text = props.t('{{author}} mentioned you in a comment in {{content}} in {{workspace}}', i18nOpts)
-          url = `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.parentContentType}/${notification.content.parentId}`
-        } else {
-          icon = 'fa-at'
-          text = props.t('{{author}} mentioned you in {{content}} in {{workspace}}', i18nOpts)
-          url = `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.type}/${notification.content.id}`
+    if (entityType === TLM_ENTITY.CONTENT) {
+      if (eventType === TLM_EVENT.created && contentType === TLM_SUB.COMMENT) {
+        return {
+          icon: 'comments-o',
+          text: props.t('{{author}} commented on {{content}} in {{workspace}}', i18nOpts),
+          url: `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.parentContentType}/${notification.content.parentId}`
         }
-        break
-      default:
-        icon = 'fa-bell'
-        text = `${notification.author} ${notification.type}`
-        url = notification.content
-          ? `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.type}/${notification.content.id}`
-          : '/ui'
+      }
+
+      const url = `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.type}/${notification.content.id}`
```

Comment:
```suggestion
      const urlContent = `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.type}/${notification.content.id}`
```
For all the urls in this file you should use the PAGE constants from helper.js